### ISSUE TITLE
[Bug] Frosted glass effect overlays where it shouldn't

### ISSUE THREAD

**Issue by CurioInventorium** [tsunamayo/Starship-EVO#2961]
### Short Description of the issue:
I placed a screen down in front of a frosted glass pane, and the frosting effect applies itself to the screen and any lettering on it.
I think it's being applied to any transparent blocks in front of it too, as the placeholder for selected block/item on the hotbar is frosted too.
...

### Steps to reproduce the issue:
Place down a frosted window and then put a screen, or possibly another block which has some transparency in front of it.

...

### Environment
- Starship EVO version: 20w37a
- Graphics Card: Geforce RTX 2060

### Additional information:
Attached picture shows a screen being partially obscured by the frost effect, even though it has been placed in front of the frosted glass.
Also to note: By the centre cursor is the small triangle block placement being frosted out too.
_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5218483/output_log.txt)
![Frosted Glass Bug](https://user-images.githubusercontent.com/51359034/93087903-1cacd100-f691-11ea-8379-e21de4cc8ec8.jpg)

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`

**Comment by tsunamayo** [comment]
Good catch, thanks for reporting this. Cheers

**Comment by Calvert4096** [comment]
Same, here's another example where I was trying to put up a screen with some text... I found a quick fix though.

Flat tile between screen and frosted glass:
![image](https://user-images.githubusercontent.com/65469633/95776927-1868d900-0c7a-11eb-9cde-885e50318593.png)

Same setup except flat tile in the middle has been removed:

![image](https://user-images.githubusercontent.com/65469633/95776991-333b4d80-0c7a-11eb-8dff-1582270f1b16.png)

**Comment by tsunamayo** [comment]
Yes indeed, this one is gonna takes a bit more time to fix - it would require a pretty big refactor of the blur pipeline. Thanks
